### PR TITLE
fix(ytmusic-borders): change colors 

### PR DESCRIPTION
**Pull request by jh-devv**
Made in favor of #37 

I think these changes are more in line with the policies regarding ["we dont change the appearance, only the colors"](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md#opinionated-changes). 

Before:
image image image

After:
image image image

### CODE REVIEW

**Comment by jh-devv** [comment]
Still don't know what the radius of that account applet should be? Probably 5 like it was?

File: youtubemusic.tera [hunk 1]
```diff
@@ -1194,7 +1193,12 @@ ytd-active-account-header-renderer.style-scope.ytd-multi-page-menu-renderer
 
 ytd-multi-page-menu-renderer.style-scope.ytmusic-popup-container
 {
-  border: 1px solid var(--ctp-accent) !important;
+  border: 1px solid var(--ctp-surface0) !important;
+}
+
+ytd-multi-page-menu-renderer
+{
+  border-radius: 2px !important;
```
Comment (jh-devv) [inline youtubemusic.tera:1201]:
```suggestion
  border-radius: 5px !important;
```
https://github.com/catppuccin/youtubemusic/pull/38#issuecomment-2168491748

Comment (kerichdev) [reply youtubemusic.tera:1201]: by popular vote thats gotta be 5

**Comment by kerichdev** [comment]
let me know when its ready for a merge :heart:

**Comment by jh-devv** [comment]
> let me know when its ready for a merge :heart: 

@kerichdev 
There is one misc change I have in my mind though, just a small color correction to the "usage popups" you might get?

I am on vacation currently so can't add that myself :3

I can probably just make an extra PR for it?

So yeah, can merge

**Comment by kerichdev** [comment]
sure, an extra pr later on sounds good - i don't have lots of time for development myself these days

thanks for the contributions!!

**Pull request merged**